### ISSUE TITLE
Feature request: search box

### ISSUE THREAD

**Issue by gasdia73** [snark/jumpcut#100]
It would be extremely useful to have a search box (like in kde clipboard) for those like me that have hundreds of texts copied, cause having to look for the right text can be a pain.

**Comment by petered** [comment]
Yes!  And to add to this - the ability to "name" or "bookmark" pastes so they persist past the max number and are easily searchable by that name (This is particularly handy for urls, which often don't contain the text that you remember them by)
